spec.bs

```diff
@@ -3990,6 +4007,55 @@ at the expense of some utility.
   </wpt>
 </div>
 
+<h3 id=credentialless-monkeypatch>Iframe credentialless</h3>
+
+The [[!IFRAME-CREDENTIALLESS]] specification defines a new object, the [=page credentialless 
+nonce=]. At a high level, the [=fenced frame config instance/partition nonce=] serves the same
+purpose as the [=page credentialless nonce=] (partitioning storage and network), but scoped to each
+fenced frame rather than to each credentialless iframe. The [=fenced frame config instance/
+partition nonce=] is also used to revoke network access in fenced frames. Perform the following
+monkeypatches to the [[!IFRAME-CREDENTIALLESS]] spec.
+
+Add the following algorithm:
+
+<div algorithm>
+  To <dfn for="browsing context">compute the effective partition nonce</dfn> given a boolean 
+  |credentialless| and [=fenced frame config instance/partition nonce=]-or-null
+  |newFencedFrameNonce|:
+
+  1. If |credentialless| is true, return the associated [=browsing context=]'s [=top-level
+     browsing context=]'s [=page credentialless nonce=].
```

Comment:
Note that the usage of "top-level browsing context" here will be fenced — is that intentional? That's because this specification does not override HTML's definition of top-level browsing context, which just iteratively reaches upward via the "parent" property, never the "unfenced parent" property.

If you want the unfenced version, you'll need to grab a [navigable's top-level traversable](https://wicg.github.io/fenced-frame/#navigable-top-level-traversable) and get its browsing context and grab its page credentialless nonce.

I just wanted to raise this in case it was unexpected. WDYT?

It seems like this algorithm prefers:
 1. A "fenced" page redentialless nonce, when we're in a credentialless iframe
 2. A "new" fenced frame nonce, when it exists
 3. The "old" navigated-away-from browsing context's fenced frame partition nonce, when we're in a fenced frame whose browsing context is apparently being navigated away from. (Is that right?)

Actually (3) above is a little confusing or surprising to me. Below on `L4031` we're always grabbing the current browsing context's fenced frame config instance's nonce, even when said browsing context is I guess being navigated away from? Am I missing something?